packages/thirdweb/src/utils/extensions/drops/fetch-proofs-for-claimers.ts

```diff
@@ -29,7 +32,26 @@ export async function fetchProofsForClaimer(options: {
     client: ThirdwebClient;
     tokenDecimals: number;
   }) => Promise<Hex>;
-}): Promise<OverrideProof | null> {
+};
+
+/**
+ * Fetch the claim proofs for a claimer's wallet address. Works with all thirdweb Drop contracts.
+ * @example
+ * ```ts
+ * import { fetchProofsForClaimer } from "thirdweb/utils";
+ *
+ * const proofs = await fetchProofsForClaimer({
+ *   contract: dropContract,
+ *   claimer: "0x...",
+ *   merkleTreeUri: "...",
+ *   tokenDecimals: 18,
+ * });
+ * ```
+ * @utils
+ */
+export async function fetchProofsForClaimer(
+  options: ProofsForClaimerParams,
```

Comment:
i wonder if its a bit hard to use for ppl. they need to understand how to get the merkleTreeUri and what tokenDecimals is for.

do we really need to expose this? I think they were fine with just using getClaimerParams no?

Reply:
Re-reading the thread, I realized you have reached out and helped them so I will undo this change ^